lib/active_model_serializers/adapter/base.rb

```diff
@@ -51,6 +51,20 @@ def include_meta(json)
         json[meta_key] = meta if meta
         json
       end
+
+      def translate_key_casing!(value, serialization_context)
```

Comment:
it would give a place for users to add their own transformations if they wish. Rather than overwriting translate_key_casing, they write their own transform method -- without losing or having to re-implement the 3 you have. I could see this being especially useful, for APIs that have clients that don't all expect the same casing.